internal/controller/operatorconfigmap_controller.go

```diff
@@ -950,3 +1002,71 @@ func (c *OperatorConfigMapReconciler) deleteDelegatedCSI() error {
 	}
 	return nil
 }
+
+// TODO: Remove it in 4.20 release
+func (c *OperatorConfigMapReconciler) getCsiCustomTolerations() ([]corev1.Toleration, []corev1.Toleration, error) {
+	rookCephOperatorConfigName, csiPluginTolerationKey, csiProvisionerTolerationKey := "rook-ceph-operator-config", "CSI_PLUGIN_TOLERATIONS", "CSI_PROVISIONER_TOLERATIONS"
```

Comment:
If you need to initialise with values, especially if the var names and the values are long, use seperate lines for each. It this case it would even be preferable to use local constants instead of vars 
```suggestion
    const (
         rookCephOperatorConfigName = "rook-ceph-operator-config"
         csiPluginTolerationKey = "CSI_PLUGIN_TOLERATIONS"
         csiProvisionerTolerationKey  = "CSI_PROVISIONER_TOLERATIONS"
     )
```